### PR TITLE
refactor: issue wizard loading

### PR DESCRIPTION
**Pull request by nud3l**

### CODE REVIEW

File: src/common/types/issue.types.ts [hunk 1]
```diff
@@ -79,4 +79,6 @@ export interface IssueState {
   issueRequests: Map<string, IssueRequest[]>;
   // TODO: remove
   vaultIssues: VaultIssue[];
+  // issue period in seconds
```
Comment (anton-karlovskiy) [inline src/common/types/issue.types.ts:82]: `// Issue period in seconds`

File: src/pages/app/issue/enter-btc-amount.tsx [hunk 2]
```diff
@@ -73,6 +74,25 @@ function EnterBTCAmount() {
   const dispatch = useDispatch();
   const { t } = useTranslation();
 
+  useEffect(() => {
+    const fetchData = async () => {
+      if (!polkaBtcLoaded) return;
+
+      try {
+        // set issue period
```
Comment (anton-karlovskiy) [inline src/pages/app/issue/enter-btc-amount.tsx:82]: `// Set issue period`

File: src/pages/app/issue/enter-btc-amount.tsx [hunk 3]
```diff
@@ -73,6 +74,25 @@ function EnterBTCAmount() {
...
+        const issuePeriodInBlocks = await window.polkaBTC.issue.getIssuePeriod();
+        const issuePeriod = new BN(issuePeriodInBlocks.toString()).mul(new BN(constants.BLOCK_TIME)).toNumber();
+        dispatch(updateIssuePeriodAction(issuePeriod));
+      } catch (error) {
+        console.log(error);
```
Comment (anton-karlovskiy) [inline src/pages/app/issue/enter-btc-amount.tsx:87]: `console.log(error.message);`

**Comment by anton-karlovskiy** [comment]
LGTM!

File: src/pages/app/issue/modal/payment-view.tsx [hunk 4]
```diff
@@ -16,30 +15,23 @@ type PaymentViewProps = {
...
-
-    fetchData();
-  }, [polkaBtcLoaded, props.request.creation]);
+    try {
+      // seconds left = request.timestamp + issue period - current time
+      const unixTimestamp = Math.floor(new Date(props.request.timestamp).getTime() / 1000);
+      const secondsLeft = unixTimestamp + issuePeriod - Math.floor(Date.now() / 1000);
+      setLeftSeconds(secondsLeft);
+    } catch (e) {
+      console.log(e);
```
Comment (anton-karlovskiy) [inline src/pages/app/issue/modal/payment-view.tsx:32]:
```
} catch (error) {
  console.log('[PaymentView] error.message => ', error.message);
}
```

FYI: I'd like us not to use abbreviations (e.g. e => error explicitly it's also mentioned in Google JS standards). :)

File: src/common/types/issue.types.ts (hunk 1 above)
Comment (nud3l) [reply src/common/types/issue.types.ts:82]: I think we should agree on a common standard. From my perspective, it's fine to not capitalize in the code comments. Is there a particular reason we should do that?

Comment (anton-karlovskiy) [reply src/common/types/issue.types.ts:82]: @nud3l 
FYI:
I checked the doc and I think it's a convention to capitalize the comments. https://google.github.io/styleguide/jsguide.html#formatting-comments
I remember setting this as a linting rule but too strict rules sometimes make things complex and lose flexibility. :)

**Pull request merged**